onnxruntime/python/tools/transformers/test/test_optimizer.py

```diff
@@ -326,10 +326,16 @@ def test_huggingface_dialogpt_fusion(self):
     def test_huggingface_bart_fusion(self):
         self._test_optimizer_on_huggingface_model("facebook/bart-base", [0, 0, 0, 0, 12, 2, 30])
 
-    def test_bert_base_cased_from_tf(self):
-        self._test_optimizer_on_tf_model("bert-base-cased", [1, 12, 0, 0, 12, 0, 24], 1)
-        self._test_optimizer_on_tf_model("bert-base-cased", [1, 12, 0, 0, 12, 0, 24], 2)
-        self._test_optimizer_on_tf_model("bert-base-cased", [1, 12, 0, 0, 12, 0, 24], 3)
+    def test_huggingface_bert_base_cased_from_tf2onnx(self):
+        self._test_optimizer_on_tf_model("bert-base-cased", [0, 12, 0, 0, 0, 0, 25], 1)
+        self._test_optimizer_on_tf_model("bert-base-cased", [0, 12, 0, 0, 0, 0, 25], 2)
+        self._test_optimizer_on_tf_model("bert-base-cased", [0, 12, 0, 0, 0, 0, 25], 3)
+
+    def test_huggingface_distilgpt2_from_tf2onnx(self):
+        self._test_optimizer_on_tf_model("distilgpt2", [0, 0, 0, 0, 0, 12, 1], 1)
+
+    def test_huggingface_albert_from_tf2onnx(self):
+        self._test_optimizer_on_tf_model("albert-base-v1", [0, 0, 0, 0, 0, 0, 25], 1)
```

Comment:
maybe to update the test? according to the description..

Reply:
The reason I have not added them is that the results of those models are slightly bigger than our threshold compared to the original model. Is it ok if I increase the "rtol" in "validate_onnx_model()" to 1e-3?

Reply:
Maybe it's better to temporarily disable the validation check like https://github.com/microsoft/onnxruntime/blob/06928e6160838fb3521b3cfae4a9f6ba9608b7df/onnxruntime/python/tools/transformers/test/test_optimizer.py#L305 so that we keep track of which model have bigger diffs
If the error of optimized model increases by one order of magnitude compared with the original one, I think we might need to take a closer look